config/routes.rb

```diff
@@ -1,6 +1,8 @@
 # frozen_string_literal: true
 
 Rails.application.routes.draw do
+  resources :errors, only: [:show]
```

Comment:
This is inside the `root.rb` file, no?

Reply:
Just saw you extracted from there. Any special reason?

Reply:
This was like this before, but we need this one to be the very first 😅

Reply:
Moving the `root` to be the first in the array below doesn't work? Also need to move this one to be the first inside the root file. Just guessing...

Reply:
Done